### ISSUE TITLE
Display manually entered issue numbers on billing report

### ISSUE THREAD

**Issue by RachelBaram** [Gizra/productivity#567]
## Current behavior
When time is reported manually on a project and an issue number is also submitted

![live_gizra_productivity](https://user-images.githubusercontent.com/14312543/40106029-55a50dc6-58fd-11e8-8246-98fe11d2413e.png)

The tracking report reflects the issue number:

![unfpa_co_ro_-_qa_lta_burn_table___live_gizra_productivity](https://user-images.githubusercontent.com/14312543/40105878-ecf9bf9c-58fc-11e8-8add-9a8cb52be4b3.png) 

The billing report does not reflect the issue number:
![unfpa_co_ro_-_qa_lta___live_gizra_productivity](https://user-images.githubusercontent.com/14312543/40105921-09a540b2-58fd-11e8-91ef-99403bade285.png)

## Desired behavior
When time is reported manually on a project and an issue number is also submitted, the billing report reflects the issue number instead of N/A.  

Note that there is no need to link the reported time against a Github Issue or PR.  For example, in UNFPA, we will be using Jira issue numbers

**Comment by bricel** [comment]
Pushed to live.
![image](https://user-images.githubusercontent.com/165644/40351928-463e4da8-5db6-11e8-83f5-d8344e1bf26f.png)

**Comment by RachelBaram** [comment]
For QA need to check the following workflows specifically in UNFPA:
1. Review or development was done in a UNFPA PR > automatic trigger of line in Productivity > Employee can enter Jira Issue number (it should be blank) along with the hours reported>Billing report shows Jira Issue number
2. QA or Management can open a new line to report their time, include a Jira Issue number > Billing report shows Jira issue number

@liatsade @noamoss - who should check this?

**Comment by liatsade** [comment]
@RachelBaram Ludmila can check this but I don't think she has permission to see the billing report. Do we have test env for this?

**Comment by RachelBaram** [comment]
I don't think we need to make this complicated including getting permissions and test environment.  It should be one or both of you doing this.  You have admin access to Productivity and access to UNFPA Github PR's.
